gen/dcos-config.yaml

```diff
@@ -211,9 +211,57 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
+          olddir /var/lib/dcos/mesos/log/archive
+          maxsize 256M
+          rotate {{ mesos_log_retention_count }}
+          copytruncate
+          postrotate
+              find /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
```

Comment:
This has nothing to do with mesos-fetcher, it has everything to do with not having a segfaulting logrotate. 

The general plan was (num log files) + 7, so that you could have some temporary user files in the log directory. Right now if I configure for lots and lots of logs, this will cap you at 14 files max or delete all the things.